apps/angular/src/app/features/home/anime-catalog/anime-catalog.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-container
+	*ngIf="{
+		animePage: animePage$ | async,
+		isLoading: isLoading$ | async,
+		filterParams: filterParams$ | async,
+		sortParams: sortParams$ | async
+	} as data"
+>
+	<camp-search-filter-form
+		(typeChange)="onSelectionChange($event)"
+		(searchChange)="onSearchChange($event)"
+		[search]="data.filterParams?.search ?? ''"
+		[selectedType]="data.filterParams?.type ?? null"
+	></camp-search-filter-form>
```

Comment:
You can use self-closing tags.
```suggestion
	<camp-search-filter-form
		(typeChange)="onSelectionChange($event)"
		(searchChange)="onSearchChange($event)"
		[search]="data.filterParams?.search ?? ''"
		[selectedType]="data.filterParams?.type ?? null"
	/>
```